aiserver.py

```diff
@@ -8675,10 +8672,11 @@ def UI_2_back(data):
 def UI_2_redo(data):
     if koboldai_vars.aibusy:
         return
-    if len(koboldai_vars.actions.get_current_options()) == 1:
-        koboldai_vars.actions.use_option(0)
+    if koboldai_vars.debug:
+        print("redo")
```

Comment:
Probably better to use `logger.debug` here

Reply:
it's something UI2 does. it's everywhere 😅

Reply:
Probably some old legacy code, i'll have to remember to convert that :P
either way for new code logger should probably be used, it gives more debug info and is prettier. plus if we dont use it, whats the point of having it?